sass/directives/_carousels.scss

```diff
@@ -105,7 +105,10 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  text-align: center;
+  line-height: 28px;//this is for vertically centering arrows in ::after
+  height: 30px;
```

Comment:
height and width - either use the `$section-padding` variable, which is contextually relevant, or use `em`s to keep this consistent with the rest of our sizings